lib/upgradable-multisig/multisig/utils.ak

```diff
@@ -0,0 +1,62 @@
+use aiken/list
+use aiken/transaction.{Input, Output}
+use aiken/transaction/credential.{Credential}
+use aiken/transaction/value.{PolicyId, Value}
+use upgradable_multisig/types.{AssetClass, MultisigDatum, PubKeyHash}
+
+pub fn get_input_by_token(inputs: List<Input>, policy_id: PolicyId) {
+  list.find(
+    inputs,
+    fn(input) { list.has(value.policies(input.output.value), policy_id) },
+  )
+}
+
+pub fn validate_multisig_datum(datum: MultisigDatum) -> Bool {
+  (list.length(datum.signers) > 1)? && (datum.threshold > 0)? && (datum.spending_limit > 2_000_000)?
```

Comment:
```suggestion
  let signers_nr = list.length(datum.signers)
  (signers_nr > 1)? && (datum.threshold > 0)? && (datum.threshold < signers_nr) && (datum.spending_limit > 2_000_000)?
```

Reply:
Changed this. Also, I want to change the condition for checking the number of signers to be greater than 0. I imagine there could be a single user who could benefit from the multi-sig.
Let me know your thoughts.